### ISSUE TITLE
`lotus-miner sectors extend` unable to renew DC sectors

### ISSUE THREAD

**Issue by beck-8** [filecoin-project/lotus#11927]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [X] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
v1.26.2
```

### Repro Steps

1. Run 'lotus-miner sectors extend --from 3903360 --to 4218480 --new-expiration 4227120 --really-do-it --max-sectors 1'
2. See error '...'
...

### Describe the Bug

```bash
root@hostname:~# lotus-miner sectors extend --from 3903360 --to 4218480 --new-expiration 4227120 --really-do-it --max-sectors 1
Extending 0 sectors: bafy2bzaceceyp6jcug4yooqkzrduaivdv6qey7b2if4sf5fsntwwsqb4xxxxx
Extending 2349 sectors: ERROR: mpool push message: GasEstimateMessageGas error: call ran out of gas
```

Problem 1: An empty structure message is generated with no parameters
Problem 2: Cannot be used normally after the second message (out of gas)

### Logging Information

```text
As above.
```

**Comment by beck-8** [comment]
This problem will occur when `--max-sectors` is less than 2349.

**Comment by beck-8** [comment]
Because the original batch operation unit is 2349, and it was CC before, the gas consumption is not that much, and the normal renewal is also more than 2349 sectors, so there is no problem. Now when renewing DC, this part is no longer satisfied, and the number needs to be less than 2349.

**Comment by beck-8** [comment]
No, some of the logic was broken when it was refactored. It's not the batch problem I mentioned earlier, I will fix it later.

**Comment by rjan90** [comment]
Closed by https://github.com/filecoin-project/lotus/pull/11928
